bindings/opencv/CMakeLists.txt

```diff
@@ -29,7 +29,12 @@ if(OPENCV_FOUND OR
     else()
         message(STATUS "The dnn example will be built (OpenCV version >= 3.4.1)")
         add_subdirectory(dnn)
-
+        if(OpenCV_VERSION VERSION_LESS "3.4.3")
+            message(WARNING "The maskR-CNN example is only available for OpenCV version >= 3.4.3! This example won't be built!")
+        else()
+            message(STATUS "The maskR-CNN example will be built (OpenCV version >= 3.4.3)")
```

Comment:
I don't think it's necessary to print this.

Reply:
For DNN example both of these messages are printed also. I think this should be consistent. Either I delete from both, either I let it this way. What do you think?

Reply:
delete both

Reply:
solved.